src/coreclr/tools/aot/ObjWriter/debugInfo/dwarf/dwarfTypeBuilder.cpp

```diff
@@ -712,8 +737,23 @@ unsigned UserDefinedDwarfTypesBuilder::GetArrayTypeIndex(
 
 unsigned UserDefinedDwarfTypesBuilder::GetPointerTypeIndex(const PointerTypeDescriptor& PointerDescriptor)
 {
+  unsigned VoidTypeIndex = GetPrimitiveTypeIndex(PrimitiveTypeFlags::Void);
+
   unsigned TypeIndex = ArrayIndexToTypeIndex(DwarfTypes.size());
-  DwarfTypes.push_back(make_unique<DwarfPointerTypeInfo>(PointerDescriptor));
+
+  // Creating a pointer to what DWARF considers Void type (DW_TAG_unspecified_type -
+  // per http://eagercon.com/dwarf/issues/minutes-001017.htm) leads to unhappines
+  // since debuggers don't really know how to handle that. The Clang symbol parser
+  // in LLDB only handles DW_TAG_unspecified_type if it's named
+  // "nullptr_t" or "decltype(nullptr)".
+  //
+  // We resort to this kludge to generate the exact same debug info for void* that
+  // clang would generate (pointer type with no element type specified).
+  if (PointerDescriptor.ElementType == VoidTypeIndex)
+      DwarfTypes.push_back(make_unique<DwarfVoidPtrTypeInfo>());
```

Comment:
Nit: Indentation

Reply:
(multiple places - 2 vs 4 spaces)

Reply:
Oh god, I was fighting VS on this everywhere. Need to find the right .editorconfig voodoo so that we can check that in.

Reply:
You can set `Default formatting style = LLVM` in Tools->Options:
https://devblogs.microsoft.com/cppblog/clangformat-support-in-visual-studio-2017-15-7-preview-1/

Reply:
The annoying part about that is that that's a global setting and I would need to turn it off depending on what project I'm working with.